grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -88,13 +88,24 @@ def section_cluster_node(outer_panels):
                             "cpu usage (total) - {{%s}} @ {{%s}}"
                             % (COMPONENT_LABEL, NODE_LABEL),
                         ),
-                        panels.target(
+                        panels.target( 
                             f"sum(rate({metric('process_cpu_seconds_total')}[$__rate_interval])) by ({COMPONENT_LABEL}, {NODE_LABEL}) / avg({metric('process_cpu_core_num')}) by ({COMPONENT_LABEL}, {NODE_LABEL}) > 0",
                             "cpu usage (avg per core) - {{%s}} @ {{%s}}"
                             % (COMPONENT_LABEL, NODE_LABEL),
                         ),
                     ],
                 ),
+                panels.timeseries_cpu(
```

Comment:
It is depend on k8s metrics, that means we need a flag to ensure it only can be used in Cloud ENV

Reply:
Let me look into that... So far I was fine with the way it is. In the risedev env it will simply show no data